xbmc/addons/AddonInstaller.cpp

```diff
@@ -259,6 +259,8 @@ bool CAddonInstaller::InstallFromZip(const std::string &path)
   if (!g_passwordManager.CheckMenuLock(WINDOW_ADDON_BROWSER))
     return false;
 
+  CLog::Log(LOGDEBUG, "CAddonInstaller: installing from zip '%s'", path.c_str());
```

Comment:
Since we logging the file path you should use `CURL::GetRedacted(path).c_str()` here.